src/context/toast-context.js

```diff
@@ -0,0 +1,22 @@
+import { createContext } from "react";
+import { useContext } from "react";
+import { useState } from "react";
+
+const ToastContext = createContext();
+
+const useToast = () => useContext(ToastContext);
+
+const ToastProvider = ({ children }) => {
+  const [toastVal, setToastVal] = useState({
+    isDisplay: "hidden",
+    select: "",
+    msg: "",
+  });
```

Comment:
I get the use of the variables but the names could be better. You can improve it.

Reply:
okay.